clustohttp.py

```diff
@@ -282,9 +282,9 @@ def add_attr(self, key, subkey, value, datatype=None, number=None):
         Returns EntityProxy of the object the attribute added
         '''
         path = "addattr?" + "key=" + key + "&subkey=" + subkey + "&value=" + value
-        if datatype:
+        if datatype is not None:
```

Comment:
If you pass an empty list, you'll get a 500 back from the clusto http api, which is I think what should happen. We can add lots of crazy validation here, but I think that it has a default arg of None, so we're just checking whether we should pass the arg based on if the api caller has explicitly called it or not